mlflow/system_metrics/metrics/cpu_monitor.py

```diff
@@ -20,8 +20,4 @@ def collect_metrics(self):
         )
 
     def aggregate_metrics(self):
-        metrics = {}
-        for name, values in self._metrics.items():
-            if len(values) > 0:
-                metrics[name] = round(sum(values) / len(values), 1)
-        return metrics
+        return {k: round(sum(v) / len(v), 1) for k, v in self._metrics.items() if v}
```

Comment:
I put check `len(values) > 0` because i saw that sometimes the value is an empty list, for which case checking `if v` is not equivalent. 

Actually checking `if v` is redundant, `v` should always exist if `k` is there. I think we can delete `if v` here, and try running unit test `pytest tests/system_metrics/` a few times to see if all tests pass.